e2e/test/Makefile

```diff
@@ -2,6 +2,7 @@ $(GOPATH)/bin/ginkgo:
 	GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
 
 ginkgo_test: $(GOPATH)/bin/ginkgo
+	GO111MODULE=on
```

Comment:
This won't work as the statement is only effective on this particular line. You can just leave a comment that this target expects the `GO111MODULE=on`. If you decide to enable it by default, you can do something like this:
```
diff --git a/e2e/test/Makefile b/e2e/test/Makefile
index 7f4a00d..edb2574 100644
--- a/e2e/test/Makefile
+++ b/e2e/test/Makefile
@@ -1,3 +1,5 @@
+export GO111MODULE=on
+
 $(GOPATH)/bin/ginkgo:
        GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
 
```

Reply:
okay